src/styles/components/_card.scss

```diff
@@ -0,0 +1,65 @@
+@use '../abstracts' as *;
+
+// Placeholder to provide base card layout
+%card-wrapper {
```

Comment:
I think we have fixed this in a prvious PR. ASked you to use the card directly in html instead of a partial class